test/e2e/mmi/README.md

```diff
@@ -0,0 +1,48 @@
+# Playwright MMI E2E Testing
+
+This directory contains a subset of MMI end-to-end tests created using [Playwright](https://playwright.dev/).
+
+## Setup
+
+To prepare your environment for the MMI E2E test suite, follow these steps:
+
+1. Create a `.env` file in this directory to store the configuration parameters for the MMI E2E test suite. Please consult with your MMI colleagues to obtain these values.
+
+2. Generate a specific MMI build:
+
+    2.1. Add the `.metamaskrc` file to the root directory. This file is necessary for generating the build and points to the development environments.
+
+    2.2. Use the following script to generate the build:
+    ```
+    yarn dist:mmi
+    ```
+
+## Running the Tests
+
+From the root of the project, you can use the following scripts to run the tests:
```

Comment:
To run the tests, I first had to run `yarn playwright install` to download Chromium.